envs/.example.env

```diff
@@ -6,4 +6,5 @@ STREAM_ACHIEVEMENTS_FEED_TOKEN=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 STREAM_TRANSACTIONS_FEED=transactions_feed
 STREAM_TRANSACTIONS_FEED_TOKEN=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 STREAM_APPID=XXXXX
-SUPPORT_CONTACT_EMAIL=XXXX@XXXXX.XXX
\ No newline at end of file
+SUPPORT_CONTACT_EMAIL=XXXX@XXXXX.XXX
+LOOM_URL=ws://XXXXXXXXXXXX
```

Comment:
I would not put specific blockchain info in env.
This is the reason I removed QTUM from env => all is in configurables now.
For ex, for qtum, ideally everything is there: https://github.com/DiademNetwork/diadem-dapp-boilerplate/blob/master/src/configurables/blockchains/qtum/index.js
It should be same for Loom.

=> Idea is that if you want to fix a blockchain/add/remove one, all is at one place and not scattered around